components/chessboard/BoardControls.jsx

```diff
@@ -123,59 +127,62 @@ export function BoardControls({ resetDisabled }) {
   }
 
   return (
-    <>
-      <div className="panel-board-controls flex-row">
-        <div
-          className={`panel-board-control flex-column hover-dim ${backDisabled && 'disabled'}`}
+    <div className="order-1 xl:order-2 bg-darkest">
+      <div className="flex p-4">
+        <button
+          className={`flex flex-col items-center w-1/3 cursor-pointer hover:opacity-50 disabled:cursor-not-allowed disabled:opacity-50`}
           disabled={backDisabled}
           onClick={goBack}
         >
-          <button className={`material-icons panel-board-control-button ${backDisabled && 'disabled'}`}>
-            chevron_left
-          </button>
+          <SVG icon={leftArrow} />
           Back
-        </div>
+        </button>
 
-        <div
-          className={`panel-board-control flex-column hover-dim ${resumeDisabled && 'disabled'}`}
+        <button
+          className={`flex flex-col items-center w-1/3 cursor-pointer hover:opacity-50 disabled:cursor-not-allowed disabled:opacity-50`}
```

Comment:
`flex-col` is the flex direction, it should be on the root element not on every child

Reply:
the button takes children, which need to be given a direction